Assets/Scripts/Characters/Human/TriggerColliderWeapon.cs

```diff
@@ -109,8 +92,13 @@ private void OnTriggerStay(Collider collider)
             case "titanneck":
                 if (collider.gameObject.TryGetComponent(out HitBox item) && item.transform.root.TryGetComponent(out TitanBase titanBase))
                 {
-                    if(!IsBehind(item.transform.root.gameObject)) 
-                        return;
+                    if (titanBase.transform.Find("Amarture/Core/Controller_Body/hip/spine/chest/neck/head") != null)
```

Comment:
titanBase has a property called "TitanBody" which contains a set of bodypart references. transform.Find should be avoided as it is a very expensive method. This goes for both line 95 and 97.

I'm not too sure if the Female and Colossal titan have this component added yet. If they do not, then please add the TitanBody and assign their head object to the TitanBody Head field. Then verify if the Colossal and Female titan can still be damaged.

Reply:
Didn't notice that one while moving code around, nice catch! Should be fixed in the most recent commit. (Tested normal, Colossal & Female titans, all were damagable)